src/daos-serialize/daos-serialize.c

```diff
@@ -127,6 +127,8 @@ int main(int argc, char** argv)
     /* create an empty file list */
     mfu_flist flist = mfu_flist_new();
 
+    printf("daos-serialize: calling daos_init\n");
+    fflush(stdout);
```

Comment:
Were these printf messages just for debugging or do you mean to leave them in?

If they should stay, do you want to wrap them so only rank 0 prints messages?  Maybe also use and ``MFU_LOG`` call?